onnxruntime/wasm/api.cc

```diff
@@ -163,6 +166,11 @@ void OrtReleaseSessionOptions(OrtSessionOptions* session_options) {
 }
 
 OrtSession* OrtCreateSession(void* data, size_t data_length, OrtSessionOptions* session_options) {
+  // OrtSessionOptions must not be nullptr.
+  if (session_options == nullptr) {
```

Comment:
`session_options` being `nullptr` is legit.

In implementation of `CreateSessionFromArray()` if it's a nullptr it's considered a default value. should not return nullptr here.

Reply:
By the way, a very important convension is that, any WASM API returning a nullptr for any type of handle indicating that there is a underlying error. Caller is expected to call `OrtGetLastError()` as the next step. So if a failure happens and a `nullptr` is returned, the last error need to be set.

Reply:
Oh, this comes from an [older](https://github.com/microsoft/onnxruntime/blob/b98317b90713ad082c3b951270f9ecdcf6f3fa0f/onnxruntime/wasm/api.cc#L138) version of the api.cc -- removed in latest commit